docs/clientset.md

```diff
@@ -0,0 +1,49 @@
+# Clientset
+
+The Chaos Controller clientset is a structured interface that simplifies interactions with the Chaos Controller's Kubernetes resources. It featured dedicated typed clients, which enable straightforward operations on chaos resources such as Disruptions and DisruptionCrons. This document provides further details on the clientset's architecture, its typed clients, and their usage.
+
+## Clientset Generation
```

Comment:
In this section we can add a concrete example with the definition of the annotations to defined and also the command line to run WDYT? In the future we will also have to give strait forward method by using the `make` command.
``` bash
client-gen \
--input-base="github.com/DataDog/chaos-controller/api" \
--input="v1beta1" \
--clientset-name="v1beta1" \
--output-dir="$(pwd)/clientset" \
--output-pkg="github.com/DataDog/chaos-controller/clientset" \
--fake-clientset=false
```

Reply:
Yes, and even include how to download and install the rc1 version you used to generate these?

Reply:
I added a guide for clientset generation. It probably would need to be updated once the Makefile is complete.